src/vmdeditoperations.cpp

```diff
@@ -131,13 +131,16 @@ void VMdEditOperations::insertImageFromPath(const QString &p_title,
                                             int p_width,
                                             int p_height)
 {
+    QString p_destImagePath;
```

Comment:
Remove the `p_` prefix of these two variables?

Reply:
ok, I will remove now